scipy/special/_precompute/utils.py

```diff
@@ -34,5 +34,5 @@ def lagrange_inversion(a):
     b = [mp.mpf(0)]
     for k in range(1, n):
         b.append(hpower[k].coeff(x, k - 1)/k)
-    b = map(lambda x: mp.mpf(x), b)
+    b = [mp.mpf(x) for x in b]
```

Comment:
The existing behaviour is be to return a `map` object, which is a generator. The new behaviour is to return a list. Is the code that calls this function supposed to expect a `map` object or a `list`?

Reply:
since the generator is a function's return value in this case, I can reverse the change just to be on the safe side. In the other cases the generator is used to generate a list within the bounds of the same function.